Sampled.xcodeproj/project.pbxproj

```diff
@@ -30,6 +30,62 @@
 			remoteGlobalIDString = 0017ED232BF17A38008B3D76;
 			remoteInfo = Sampled;
 		};
+		1B88DEA12EC68F2F007025D8 /* PBXContainerItemProxy */ = {
```

Comment:
We don't need Vorbis in the Xcode file in the same vein as how we don't need Opus or FFmpeg.